src/test/java/com/cdancy/bitbucket/rest/features/SupportApiLiveTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.features;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import com.cdancy.bitbucket.rest.BaseBitbucketApiLiveTest;
+import com.cdancy.bitbucket.rest.domain.support.SupportZip;
+import org.testng.annotations.Test;
+
+@Test(groups = "live", testName = "SupportApiLiveTest", singleThreaded = true)
+public class SupportApiLiveTest extends BaseBitbucketApiLiveTest {
+    private String taskIdRegex = "^\\w+\\-+\\w+\\-+\\w+\\-+\\w+\\-+\\w+$";
+
+    private SupportApi api() {
+        return api.supportApi();
+    }
+
+    @Test
+    public void testSupportZipCreation() {
+        final SupportZip supportZipTask = api().createSupportZip();
+        assertIntegrity(supportZipTask);
+    }
+
+    @Test
+    public void testSupportZipStatus() {
+        SupportZip supportZipTask = api().createSupportZip();
+        supportZipTask = api().getSupportZipStatus(supportZipTask.taskId());
```

Comment:
Lets also test the case where things fail. Meaning we can pass in a taskId which does not exist and the returned `Errors` list should be populated with objects.